t/plugin/response-rewrite.t

```diff
@@ -699,3 +699,50 @@ X-A: 127.0.0.1
 X-B: from 127.0.0.1 to 127.0.0.1:1980
 --- no_error_log
 [error]
+
+
+
+=== TEST 25: set empty body
```

Comment:
I run your test locally, and it can be passed even without newly added modification.

Reply:
Are there any relevant examples? We tested and found that the original code would go wrong when running this case

Reply:
@spacewander  hi you can see https://github.com/Hazel6869/apisix/runs/8244062115?check_suite_focus=true